docs/api/rate-limits.md

```diff
@@ -38,7 +38,7 @@ The following tables list the rate limits for the NuGet.org API.
 > [!Note]
 > We recommend using NuGet.org's [V3 search APIs](search-query-service-resource.md) as it is not rate limited currently. For V1 and V2 search APIs, the following limits apply:
 
-| API | Limit Type | Limit Value | API usecase |
+| API | Limit Type | Limit Value | API use case |
```

Comment:
Should we have consistent casing for the headings? Either have `Limit type` and `Limit value`, or, change your heading to `API Use Case`. Same comment for line 50